python/lsst/dax/data_generator/generator.py

```diff
@@ -169,27 +172,30 @@ def make_chunk(self, chunk_id, edge_width=0.017, edge_only=False, return_pregene
             generated_data_per_box = []
             boxes = self._make_boxes(chunk_id, edge_width=edge_width,
                                      edge_only=edge_only)
-            chunk_center = SkyCoord(ra_center, dec_center, frame="icrs", unit="deg")
 
             for box_n, box in enumerate(boxes):
                 assert(box.area() > 0)
                 box_rowcount = int(chunk_density * box.area())
                 if(box_rowcount == 0):
                     continue
                 unique_box_id = chunk_id*8 + box_n
+                box_center_ra = (box.raA + box.raB)/2.0
+                box_center_dec = (box.decA + box.decB)/2.0
+                box_center = SkyCoord(box_center_ra, box_center_dec, frame="icrs", unit="deg")
                 output = self._generate_table_block(box, column_generators,
-                                                    chunk_center=chunk_center,
+                                                    chunk_center=box_center,
```

Comment:
I ran into an issue with the trimming process above missing some of the points in the north and south boxes due to the chunk center being different than the box center. I can change the name.

Reply:
It looks like that might be because you do the trimming based on the box center, but the actual calculation was done on the chunk center. You have access to the chunk center in the function doing the trimming, wouldn't using that solve the inconsistency?

Reply:
The function really operates on boxes, so sticking with box boundaries and box centers seems more appropriate.